README.md

```diff
@@ -0,0 +1,155 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Back-End-Link](#backend-link)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 React Front-End <a name="about-project"></a>
+
+**React Front-End** seamlessly retrieves and displays a variety of random greetings sourced from a Rails API, offering users a dynamic and engaging experience with each visit.
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.pgsclusters.com/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Displays random greetings**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Backend Link <a name="backend-link"></a>
+
+- [Live Demo Link](https://github.com/CaptainBawa/Hello-Rails-Back-End)
```

Comment:
- [ ] Kindly name this link correctly. It should be something like backend link instead of live demo link, proper documentation.